sdk/tables/aztables/cosmos_patch_transform_policy.go

```diff
@@ -0,0 +1,27 @@
+// +build go1.13
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package aztables
+
+import (
+	"net/http"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+)
+
+// CosmosPatchTransformPolicy transforms PATCH requests into POST requests with the "X-HTTP-Method":"MERGE" header set.
+type CosmosPatchTransformPolicy struct{}
```

Comment:
Wondering if this should have a ctor...like NewCosmosPatchTransformPolicy? To match the policies in azcore. This is probably a bigger discussion that may require both Jeff and Joel's input as well. In my opinion it's a bit odd to call ctors for all the other and not just for this one..

Reply:
I had the same thought - the only reason I didn't is that there is no config, so a standard instance with no arguments is all is needed. I'm fine to change it if we decided that it makes sense.